test_ffi/tests/integration_tests.rs

```diff
@@ -66,10 +66,10 @@ fn basic() {
     5\n\
     5\n\
     579\n\
-    8589934590\n\
-    -8589934590\n\
-    8589934590\n\
-    -8589934590\n\
+    8589934590n\n\
```

Comment:
It's a bit confusing that now we will return 64 bit integers below the MAX_SAFE_INTEGER limit as BigInts in Fast API calls, even though the equivalent slow call would return a plain number. Should the unpacking maybe check against the limit and return a converted plain number if it's safe to do so? That way Fast API return values would not start causing deopts in calls that reuse those values. (Though, we could also do the safe BigInt to number conversion on the JS side before calling.)

Reply:
Fixed